src/components/ApplicationForm/BasicInfo.js

```diff
@@ -0,0 +1,375 @@
+        canCreateNewOption
+        isValid
+      ></Dropdown>
+    </FormSpacing>
+
+    <FormSpacing>
+      <QuestionHeading>question 07</QuestionHeading>
+      <H1 size="1.5em">What is your current or intended major?</H1>
+      <Dropdown
+        options={majors}
+        placeholder="Enter your major"
+        isSearchable
+        formatCreateLabel={inputValue => `My major is not listed, use "${inputValue}"`}
+        label={formInputs.major}
+        onChange={e =>
+          onChange({
+            ...formInputs,
+            major: e.label,
+          })
+        }
+        emptySearchDefaultOption="Start typing to search"
+        canCreateNewOption
+        isValid
+      ></Dropdown>
+    </FormSpacing>
+
+    <FormSpacing>
+      <QuestionHeading>question 08</QuestionHeading>
+      <H1 size="1.5em">What is your current level of education?</H1>
+      <Dropdown
+        options={educationOptions}
+        placeholder="Level of Education"
+        isSearchable={false}
+        value={formInputs.educationLevel}
+        onChange={inputValue =>
+          onChange({
+            ...formInputs,
+            educationLevel: inputValue.value,
+          })
+        }
+        isValid
+      ></Dropdown>
+    </FormSpacing>
+
+    <FormSpacing>
+      <QuestionHeading>question 09</QuestionHeading>
+      <H1 size="1.5em">What is your graduation year?</H1>
+      <Dropdown
+        options={graduationOptions}
+        placeholder="Graduation Year"
+        isSearchable={false}
+        value={formInputs.graduation}
+        onChange={inputValue =>
+          onChange({
+            ...formInputs,
+            graduation: inputValue.value,
+          })
+        }
+        isValid
+      ></Dropdown>
+    </FormSpacing>
+
+    <FormSpacing>
+      <QuestionHeading>question 10</QuestionHeading>
+      <H1 size="1.5em">How many hackathons have you attended (both online and in-person)?</H1>
+      <Dropdown
+        options={hackathonOptions}
+        placeholder="Number of Hackathons"
+        isSearchable={false}
+        value={formInputs.hackathonsAttended}
+        onChange={inputValue =>
+          onChange({
+            ...formInputs,
+            hackathonsAttended: inputValue.value,
+          })
+        }
+        isValid
+      ></Dropdown>
+    </FormSpacing>
+
+    <FormSpacing>
+      <QuestionHeading>question 11</QuestionHeading>
+      <H1 size="1.5em">
+        How do you want to contribute at nwHacks? Please select the category that you're strongest
+        in.
+      </H1>
+      <Select
+        type="radio"
+        label="Developer"
+        checked={formInputs.contributionRole === 'Developer'}
+        onChange={() => onChange({ ...formInputs, contributionRole: 'Developer' })}
+      ></Select>
+      <Select
+        type="radio"
+        label="Designer"
+        checked={formInputs.contributionRole === 'Designer'}
+        onChange={() => onChange({ ...formInputs, contributionRole: 'Designer' })}
```

Comment:
nit: can these be lowercase?
```suggestion
        checked={formInputs.contributionRole === 'developer'}
        onChange={() => onChange({ ...formInputs, contributionRole: 'developer' })}
      ></Select>
      <Select
        type="radio"
        label="Designer"
        checked={formInputs.contributionRole === 'designer'}
        onChange={() => onChange({ ...formInputs, contributionRole: 'designer' })}
```